packages/ui/package.json

```diff
@@ -42,7 +42,7 @@
   "dependencies": {
     "lodash": "4.17.21",
     "style-dictionary": "3.7.0",
-    "xstate": "^4.30.6",
+    "xstate": "^4.33.0",
```

Comment:
For my knowledge, why not pin the version?

Reply:
Good question!

This is to avoid package duplication: Gatsby depends on `xstate@^4.x.x`, so if we pin the version, customer will get duplicated versions of xstate. And we found that duplicated xstate causes runtime errors.

More details here! https://github.com/aws-amplify/amplify-ui/pull/1545

Reply:
Got it, thanks!